clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/MoveExecutor.scala

```diff
@@ -175,16 +183,19 @@ class MoveExecutor[TI <: TransferIndex](moveCommand: MoveCommand[TI])
           }
         _ <- Future
           .sequence(googleDeletes)
-          .recover {
-            case ex =>
+          .map((moves: Iterable[Either[URI, Unit]]) => {
+            if (moves.forall(_.isRight)) {
+              logger.info("All objects were successfully deleted in gcloud.")
+            } else {
+              val notDeleted = moves.filter(_.isLeft)
               throw new RuntimeException(
                 s"""The old files associated with $prettyKey were not able to be deleted.
-                   |Please manually delete the old files at:
-                   |${oldPaths.mkString(",\n")}
-                   |If this can't be done, please contact the Green Team at ${ClioClientConfig.greenTeamEmail}""".stripMargin,
-                ex
+                 |Please manually delete the old files at:
+                 |${notDeleted.mkString(",\n")}
+                 |If this can't be done, please contact the Green Team at ${ClioClientConfig.greenTeamEmail}""".stripMargin
```

Comment:
If you cannot delete the files, ...
OR
Please tell the Green Team at ... if you cannot delete the files.